tools/ALARAJOYWrapper/README.md

```diff
@@ -0,0 +1,67 @@
+# FENDL3 Preprocessor with NJOY Wrapping for ALARA (*ALARAJOYWrapper*)
+
+This preprocessor is designed to update ALARA's data input capabilities for updated FENDL3 data formats. Unlike previous versions of [FENDL](https://www-nds.iaea.org/fendl_library/websites/fendl32b/) (Fusion Evaluated Nuclear Data Library) which have available groupwise cross-section data for neutron activation, which is required for ALARA's functionality, FENDL3 data is only offered as pointwise.
+
+This preprocessor uses the [NJOY 2016](https://github.com/njoy/NJOY2016) Nuclear Data Processing System's GROUPR module to convert the pointwise nuclear data to the Vitamin-J 175 group groupwise format for cross-sections and handle the processed data to be fed back to ALARA.
+
+## Dependencies
```

Comment:
I'm inclined to have these dependencies grouped into three categories:
* standard Python libraries
* well known generic python packages (e.g. Pandas, Numpy)
* domain-specific packages (NJOY, ENDFtk)